src/project/settingup_forms.md

```diff
@@ -309,4 +309,53 @@ To use the drop-down widget in the form from Input:
 
 ![slider](./input_forms_valuerelation1.png)
 ![slider](./input_forms_valuerelation2.png)
-![slider](./input_forms_valuerelation3.png)
\ No newline at end of file
+![slider](./input_forms_valuerelation3.png)
+
+## 1-N relations
+
+It is often the case that you have a set of spatial features and you want to record some parameters every now and then. For example, there is a GIS layer representing the manholes and the surveyors carry out regular inspections of the manholes. Instead of duplicating the manhole layer and recording each inspection, you can create a non-spatial table and store each inspection as a new line. An example project
+
+Another example is when you try to capture multiple photos for a single feature.
+
+In the sections below, you can see how this type of relations can be set up in QGIS and utilised in the Input app.
+
+### Project preparation in QGIS
+
+The example projects can be found:
+- [Assigning multiple inspection to a single feature](<MerginMapsProject id="documentation/forms_one-to-many-relations" />)
+- [Adding multiple photos to a single feature](<MerginMapsProject id="documentation/forms_multiple_photos" />)
+
+To configure 1-N relations in QGIS:
```

Comment:
Before showing how to set it up for a particular project, it would be worth showing the structure of the tables (what are the columns and maybe few sample rows), so it is is clear how the data are linked.